proposals/0029-cooperative-vector.md

```diff
@@ -239,6 +239,20 @@ This operation returns a vector of size `NUMo` and contains elements of type
 `TYo`. The result vector does not have an interpretation parameter, its type is
 the declared type.
 
+#### Operation Flags
+
+The operation flags are generated by OR'ing together values from these constants:
+
+```
+const uint8_t kMatVecMulFlag_UnsignedInputVector = 0x1;
```

Comment:
Can we call this `KMatVecOpFlag*` as it covers both the Mul and MulAdd operations?